blackbox/docs/cli-tools.rst

```diff
@@ -101,5 +108,111 @@ The CrateDB process can handle the following signals.
     the node is stopped, you can perform a `graceful stop`_ with the
     :ref:`DECOMMISSION <alter_cluster_decommission>` statement instead.
 
+Crate-node
+==========
+
+Managing clusters
+-----------------
+
+This section shows you how to manage CrateDB clusters from the command line.
```

Comment:
merge this into the following note block then move it after the example `--help` command